content/en/docs/concepts/overview/kubernetes-api.md

```diff
@@ -197,14 +198,19 @@ Elimination of resources or fields requires following the
 
 Kubernetes makes a strong commitment to maintain compatibility for official Kubernetes APIs
 once they reach general availability (GA), typically at API version `v1`. Additionally,
-Kubernetes keeps compatibility even for _beta_ API versions wherever feasible:
-if you adopt a beta API you can continue to interact with your cluster using that API,
-even after the feature goes stable.
+Kubernetes maintains compatibility with data persisted via _beta_ API versions of official Kubernetes APIs,
```

Comment:
Is 'official' the same as Generally Available (GA)?

Kubernetes maintains compatibility between data persisted by _beta_ versions and ...

Reply:
The use of "official Kubernetes APIs" here seems to be broader than intended. My understanding is that statement would include Gateway API and other CRD based APIs, but of course CRDs are going to follow at least slightly different versioning guidelines. Are you actually trying to cover "official in-tree Kubernetes APIs"? I don't really know the right terminology to use for that, but I think it's an important distinction to make with more CRD based APIs on the way.

Reply:
How do we translate k8s minor releases to a wholly independent CRD release cycle? Specifically this statement is wholly incompatible with Gateway API versioning where different resources will graduate to beta and GA in different releases and minor releases are not tied in anyway to Kubernetes release schedule:

```
- Beta API versions have a maximum lifetime of 3 minor releases from introduction to deprecation, and 3 minor releases from deprecation to removal.
```

Reply:
Removal works differently and is not externally tied to the k8s lifecycle in the same way for CRD-based APIs, I agree. I'll do another pass trying to clarify which aspects are specifically for built in APIs

Reply:
> Is 'official' the same as Generally Available (GA)?

No, official as in "APIs revised by Kubernetes". We can't make any guarantees APIs provided by other projects as CRDs or aggregated API servers will be well-behaved on the way from beta to stable and will keep these promises.